extensions/wikia/MyHome/SpecialWikiActivity.class.php

```diff
@@ -112,7 +112,8 @@ function execute( $par ) {
 
 		// page header: replace subtitle with navigation
 		global $wgHooks;
-		$wgHooks['BeforePageHeaderSubtitle'][] = [ $this, 'addNavigation' ];
+		$wgHooks['AfterPageHeaderPageSubtitle'][] = [ $this, 'addNavigationToPageHeader' ];
+		$wgHooks['AfterPageHeaderSubtitle'][] = [ $this, 'addCheckboxToPageHeader' ];
 
 		if ( $wgUser->isAnon() ) {
 			$this->getOutput()->setSquidMaxage( 3600 ); // 1 hour
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Move this trailing comment on the previous empty line. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS139)